packages/components/Notification/src/NotificationTokens.ts

```diff
@@ -6,6 +6,8 @@ export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme>
   ({
     backgroundColor: t.colors.background,
     color: t.colors.brandForeground1,
+    borderColor: '#FFFFFF',
```

Comment:
Hardcoding white here seems like it would show poorly on dark modes, does it look ok?

Reply:
Looks ok so far 
<img width="300" src="https://user-images.githubusercontent.com/55368679/177378670-3e7c9fd5-14d0-4827-ae6e-073f4ec6afe2.png">

Reply:
Edit: I think I misread your comment. For toasts, the border is actually supposed to be clear but white was the closest I could get.

Reply:
You can set the color to 'transparent' if it's supposed to be clear.

Reply:
(probably transparentStroke is better since then on HC it'll still draw a border)

Reply:
hmm transparentStroke adds a black border even without HC?

Reply:
That's odd. @chiuam do you know what's up here?

Reply:
It seems that black is the default color and transparentStroke either doesn't exist or is not working so it defaults to black

Reply:
Ah yeah it might not be filled out on iOS. Transparent seems ok then, though you might need to change the color when using a high contrast mode if those exist on iOS.

Reply:
It looks like on iOS, we never defined the token transparentStroke, and in appleColors.ios.ts, I just did this:
>     primaryButtonBorder: 'transparent',

So it's probably OK to just set 'transparent' where needed. We have iOS FURN token support planned out for Fy23 and can update this when that happens.